chunky/src/java/se/llbit/chunky/model/QuadModel.java

```diff
@@ -15,6 +16,43 @@
 @PluginApi
 public abstract class QuadModel implements BlockModel {
 
+  public static final Quad FULL_BLOCK_NORTH_SIDE = new Quad(
+    new Vector3(1, 0, 0),
+    new Vector3(0, 0, 0),
+    new Vector3(1, 1, 0),
+    new Vector4(0, 1, 0, 1));
+  public static final Quad FULL_BLOCK_SOUTH_SIDE = new Quad(
+    new Vector3(0, 0, 1),
+    new Vector3(1, 0, 1),
+    new Vector3(0, 1, 1),
+    new Vector4(0, 1, 0, 1));
+  public static final Quad FULL_BLOCK_WEST_SIDE = new Quad(
+    new Vector3(0, 0, 0),
+    new Vector3(0, 0, 1),
+    new Vector3(0, 1, 0),
+    new Vector4(0, 1, 0, 1));
+  public static final Quad FULL_BLOCK_EAST_SIDE = new Quad(
+    new Vector3(1, 0, 1),
+    new Vector3(1, 0, 0),
+    new Vector3(1, 1, 1),
+    new Vector4(0, 1, 0, 1));
+  public static final Quad FULL_BLOCK_TOP_SIDE = new Quad(
+    new Vector3(1, 1, 0),
+    new Vector3(0, 1, 0),
+    new Vector3(1, 1, 1),
+    new Vector4(1, 0, 1, 0));
+  public static final Quad FULL_BLOCK_BOTTOM_SIDE = new Quad(
+    new Vector3(0, 0, 0),
+    new Vector3(1, 0, 0),
+    new Vector3(0, 0, 1),
+    new Vector4(0, 1, 0, 1));
```

Comment:
Note that these quads were rotated specifically such that `TopBottomOrientedTexturedBlockModel` textures are rotated correctly on all sides. This may not apply to any full cube blocks.

Reply:
is there already a model that rotates the sides but not the top/bottom, or would that need to be created?

Reply:
As far as I understand, the Quad's orientation should each currently match the correct coordinate axis when not rotated.
I wanted to work on more utilities for block models in the future - having these constants should also be helpful for them. (And should reduce the global object count if reused in multiple models, right?)